e2e/nomostest/config_sync.go

```diff
@@ -88,16 +88,26 @@ var (
 	reconcilerPollingPeriod time.Duration
 	// hydrationPollingPeriod specifies hydration-controller polling period as time.Duration
 	hydrationPollingPeriod time.Duration
+	// helmSyncVersionPollingPeriod specifies helm sync versionr polling period as time.Duration
+	helmSyncVersionPollingPeriod time.Duration
 )
 
-// IsReconcilerManagerConfigMap returns true if passed obj is the
+// IsReconcilerManagerCMConfigMap returns true if passed obj is the
 // reconciler-manager ConfigMap reconciler-manager-cm in config-management namespace.
-func IsReconcilerManagerConfigMap(obj client.Object) bool {
+func IsReconcilerManagerCMConfigMap(obj client.Object) bool {
 	return obj.GetName() == "reconciler-manager-cm" &&
 		obj.GetNamespace() == "config-management-system" &&
 		obj.GetObjectKind().GroupVersionKind() == kinds.ConfigMap()
 }
 
+// IsReconcilerManagerConfigMap returns true if passed obj is the
+// reconciler-manager ConfigMap reconciler-manager in config-management namespace.
+func IsReconcilerManagerConfigMap(obj client.Object) bool {
```

Comment:
See the explanation in https://github.com/GoogleContainerTools/kpt-config-sync/pull/707#discussion_r1241009308